test/Serilog.Sinks.MSSqlServer.Tests/DatabaseFixture.cs

```diff
@@ -42,7 +42,7 @@ public DatabaseFixture()
         
         public void Dispose()
         {
-            DeleteDatabase();
+            //DeleteDatabase();
```

Comment:
Yes and no. Depends on whether or not you want the temporary database around to inspect after your tests finish running... the constructor deletes it before starting the tests.